tests/cpp_integration_tests/test_main_model.cpp

```diff
@@ -372,37 +372,6 @@ TEST_CASE("Test main model - individual output (asymmetric)") {
     auto const res = main_model.calculate<power_flow_t, asymmetric_t>(
         get_default_options(asymmetric, CalculationMethod::newton_raphson));
 
-    /*
-    TODO:
-    - test node
-    - test line
-    - test link
-    - test source
-    - test sym load
-    - test asym load
-    - test shunt
-    */
-
-    SUBCASE("Node, asym output") { // TODO(mgovers): p and q not tested in power_flow/dummy-test/sym_output.json
-        main_model.output_result<Node>(res, state.asym_node);
```

Comment:
This was removed but hasn't been added yet to the validation tests, right?

Reply:
1. i added `power_flow/dummy-test/asym_output.json` for the asym output; that part is now covered
2. i had a quick chat with Nitish and we decided that this is not the right location for the `p` and `q` checks